CyvoreOS/phone_numbers_extension.py

```diff
@@ -0,0 +1,270 @@
+    "+45",  # Denmark: 12 34 56 78 -> +45 12 34 56 78
+    "+372",  # Estonia: 5123 4567 -> +372 5123 4567
+    "+298",  # Faroe Islands: 212345 -> +298 212345
+    "+358",  # Finland: 9 123 4567 -> +358 9 123 4567
+    "+33",  # France: 01 09 75 83 51 -> +33 1 09 75 83 51
+    "+995",  # Georgia: 32 123 4567 -> +995 32 123 4567
+    "+49",  # Germany: (30) 1234-5678 -> +49 30 1234-5678
+    "+350",  # Gibraltar: 200 12345 -> +350 200 12345
+    "+30",  # Greece: 21 1234 5678 -> +30 21 1234 5678
+    "+299",  # Greenland: 321234 -> +299 321234
+    "+36",  # Hungary: 1 234 5678 -> +36 1 234 5678
+    "+354",  # Iceland: 123 4567 -> +354 123 4567
+    "+353",  # Ireland: (01) 234 5678 -> +353 1 234 5678
+    "+39",  # Italy: 06 6989 1234 -> +39 06 6989 1234
+    "+383",  # Kosovo: 38 123 456 -> +383 38 123 456
+    "+371",  # Latvia: 67 123 456 -> +371 67 123 456
+    "+423",  # Liechtenstein: 234 5678 -> +423 234 5678
+    "+370",  # Lithuania: (8-5) 123 4567 -> +370 8-5 123 4567
+    "+352",  # Luxembourg: 621 123 456 -> +352 621 123 456
+    "+356",  # Malta: 21 234 567 -> +356 21 234 567
+    "+373",  # Moldova: 22 123 456 -> +373 22 123 456
+    "+377",  # Monaco: 93 15 67 89 -> +377 93 15 67 89
+    "+382",  # Montenegro: 20 123 456 -> +382 20 123 456
+    "+31",  # Netherlands: 20 123 4567 -> +31 20 123 4567
+    "+389",  # North Macedonia: 2 123 456 -> +389 2 123 456
+    "+47",  # Norway: 123 45 678 -> +47 123 45 678
+    "+48",  # Poland: 22 123 45 67 -> +48 22 123 45 67
+    "+351",  # Portugal: 21 234 5678 -> +351 21 234 5678
+    "+40",  # Romania: 21 234 5678 -> +40 21 234 5678
+    "+7",  # Russia: (495) 123-45-67 -> +7 495 123-45-67
+    "+378",  # San Marino: 0549 123 456 -> +378 549 123 456
+    "+381",  # Serbia: 11 123 4567 -> +381 11 123 4567
+    "+421",  # Slovakia: 2/123 45 67 -> +421 2/123 45 67
+    "+386",  # Slovenia: 31 123 456 -> +386 31 123 456
+    "+34",  # Spain: 912 345 678 -> +34 912 345 678
+    "+46",  # Sweden: (08) 123 4567 -> +46 8 123 4567
+    "+41",  # Switzerland: (0)22 123 45 67 -> +41 22 123 45 67
+    "+90",  # Turkey: (212) 123 4567 -> +90 212 123 4567
+    "+380",  # Ukraine: 44 123 4567 -> +380 44 123 4567
+    "+44",  # United Kingdom: (20) 7946 0958 -> +44 20 7946 0958
+    "+39",  # Vatican City: 06 6989 -> +39 06 6989
+]
+
+
+def find_phone_numbers(data: str) -> List[str]:
+    """
+    Finds Phone Numbers If Exist In Data
+    """
+    rgx_possible = set()
+    logging.info("Looking For Phone Numbers")
+    matches_us = re.findall(USPHONEREG, data)
+    matches_israel = re.findall(ISPHONEREG, data)
+    matches_europe = re.findall(EUPHONEREG, data)
+    logging.debug(
+        "Regex matches - US: %d, Israel: %d, Europe: %d",
+        len(matches_us),
+        len(matches_israel),
+        len(matches_europe),
+    )
+    for match in matches_us + matches_israel + matches_europe:
+        rgx_possible.add((match.strip()).lstrip("+"))
```

Comment:
I think you want to remove all spaces, so you should use match.replace(" ", "")
Consider maybe do so for other special characters? like "(" ")" "-" and etc?